src/Contexts/FilterContext/filter-context.js

```diff
@@ -0,0 +1,27 @@
+import { createContext, useContext, useReducer } from "react";
+import { filterReducer } from "../../Reducers";
+
+const FilterContext = createContext();
+
+const FilterProvider = ({ children }) => {
+  const initialState = {
+    sortBy: undefined,
```

Comment:
we can also give null as the initial value.

Reply:
Yes we can.